monai/metrics/surface_distance.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Tuple, Union
+
+import numpy as np
+import torch
+
+from monai.transforms.croppad.array import SpatialCrop
+from monai.transforms.utils import generate_spatial_bounding_box
+from monai.utils import optional_import
+
+binary_erosion, _ = optional_import("scipy.ndimage.morphology", name="binary_erosion")
+distance_transform_edt, _ = optional_import("scipy.ndimage.morphology", name="distance_transform_edt")
+
+
+def get_mask_edges(
```

Comment:
this could be a generic utility in `monai/metrics/utils.py`? `compute_hausdorff_distance` uses the same logic as well